rllab/plotter/plotter.py

```diff
@@ -1,8 +1,10 @@
 import atexit
```

Comment:
did this file get accidentally mixed with your other commit for async plotting?

see my slack announcement about rebasing.